service/javadsl/integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -187,12 +184,9 @@ public static LagomClientFactory create(String serviceName, ClassLoader classLoa
         EventLoopGroup eventLoop = new NioEventLoopGroup();
 
         // WS
-        WSClientConfig wsClientConfig = new WSConfigParser(configuration, environment).parse();
-        AhcWSClientConfig ahcWSClientConfig = new AhcWSClientConfigParser(wsClientConfig, configuration, environment).parse();
-        new SystemConfiguration().configure(wsClientConfig);
-        DefaultAsyncHttpClientConfig.Builder configBuilder = new AhcConfigBuilder(ahcWSClientConfig).configure();
-        configBuilder.setEventLoopGroup(eventLoop);
-        WSClient wsClient = new AhcWSClient(configBuilder.build(), materializer);
+        WSClientConfig wsClientConfig = new WSConfigParser(configuration.underlying(), environment.classLoader()).parse();
+        AhcWSClientConfig ahcWSClientConfig = new AhcWSClientConfigParser(wsClientConfig, configuration.underlying(), environment.classLoader()).parse();
+        WSClient wsClient = AhcWSClient.apply(ahcWSClientConfig, scala.Option.empty(), materializer);
```

Comment:
Also, I'm not sure about this one, but it looks like the `WsClient` injected in `JavadslServiceClientImplementor` hasn't changed in your diff. So it's still using the one from Play. (not sure why they kept a `play-ahc-ws` module in the framework instead of using the Play WS Standalone dependency, you might know better than me).